### ISSUE TITLE
OrbitDB aggressive replication.

### ISSUE THREAD

**Issue by farhoud** [functionland/fula#201]
The replication mechanism for orbitDB is too simple and need new design :
- for now if aggressively replicated it self on all the box that available on the network.  

We have to consider:
- User Space.
- Replication Factor

**Comment by mehdibalouchi** [comment]
@farhoud I totally agree with the need for user spaces. They should also take dApps into account.

about the Replication Factor, I can see the high availability is necessary but promising to store an exact number of copies of a record seems to be just a barrier with no use case.

**Comment by farhoud** [comment]
Oh the real problem with our replicatore is if we have 100 box that runs box. all of there collection would be replicated which Is not really ideal. (don't forget that orbitdb is a append only with no rollup so it going to take space).

**Comment by farhoud** [comment]
@mehdibalouchi 
Did you ream my response?

**Comment by mehdibalouchi** [comment]
@farhoud If I understand your point correctly, you are saying that in order for the data protocol records to be highly available there is no need to replicate them on every box in the network. A much less replication factor must be sufficient.
I agree with that and as far as I know, orbitDB does not have a replication factor configuration.

**Comment by farhoud** [comment]
I does not have a replicator, I wrote the aggressive replicator and I think it need your attention.
